README.md

```diff
@@ -229,6 +229,85 @@ const searchResults = await searchClient.productSearch({
 
 Invalid query parameters that are not a part of the API and do not follow the `c_` custom query parameter convention will be filtered from the request and a warning will be displayed.
 
+### Custom APIs
+
+The SDK supports calling [custom APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/custom-apis.html) with a helper function, `customApiHelper.callCustomEndpoint()`.
+
+Example usage:
+
+```javascript
+import * as CommerceSdk from "commerce-sdk";
+const { customApiHelper } = CommerceSdk;
```

Comment:
Looking at how the other helpers are imported:

`import { Search, Customer, helpers, slasHelpers } from "commerce-sdk";`

Should `customApiHelper` be under the helpers object since it's a singular helper?

Reply:
Maybe `customApiHelper` should be `customApiHelpers` ?

Reply:
I believe in the isomorphic implementation, the new helper is under the `helpers` object. Do we want to keep it consistent across libs?

This is from the PR description of the isomorphic change.
```js
import {helpers} from 'commerce-sdk-isomorphic';

let response = await helpers.callCustomEndpoint({
    method: 'GET',
    headers: {
        'Content-Type': 'application/json',
        authorization: `Bearer ${access_token}`
    }
}, clientConfig, false)
```

Reply:
Updated to have callCustomEndpoint under `helpers`